packages/odyssey-babel-plugin/src/compileFactory.ts

```diff
@@ -47,10 +47,10 @@ export default function compileFactory(): Compile {
       delete result.message.warning;
     }
 
-    worker.unref();
-
     return result.message;
   };
 
+  worker.unref();
+
   return compile;
 }
```

Comment:
This may look weird, but it was like this in the last PR. I'd changed it, then realized the callback never got called which is why Babel never stopped. It was breaking a lot of stuff until I reverted my previous change.